packages/corejs/src/monitoring/index.tsx

```diff
@@ -0,0 +1,128 @@
+import React, { createContext, useContext, useEffect, useMemo } from 'react';
+import { Primitive } from '@sentry/types';
+import { CaptureConsole } from '@sentry/integrations';
+import { Integrations } from '@sentry/tracing';
+import { Platform } from 'react-native';
+import * as Sentry from 'sentry-expo';
+import * as SentryR from '@sentry/react';
+import Constants from 'expo-constants';
+import PropTypes from 'prop-types';
+
+import { MonitoringContextType } from './types';
+
+const tracingOrigins = ['localhost', 'cna.dev.monk.ai', 'cna.staging.monk.ai', 'cna.preview.monk.ai', 'cna.monk.ai'];
```

Comment:
What do the tracing origins do exactly ? Because I don't think we should hard code them like that. We should allow users to pass them as configuration parameters for the useMonitoring hook. There are A LOT of other URLs in which the SDK is used. Just in Monk alone there are at least 20 different URLs where the SDK is used, and this is without taking the Clients into account.

By the way there are some mistakes in the URLs here because for instance the stagng URL is the following : `cna-staging.dev.monk.ai`